wavpackdll/wavpackdll.rc

```diff
@@ -7,7 +7,7 @@
 //
 // Generated from the TEXTINCLUDE 2 resource.
 //
-#include "afxres.h"
+#include <windows.h>

```

Comment:
`winresrc.h` wolud be better

Reply:
Applied in e8ac75f9ff9d4564117b6204323f3ed4845e910b